app/portainer/views/home/home.html

```diff
@@ -40,7 +40,7 @@
   <div class="col-sm-12">
     <endpoint-list
       title-text="Endpoints" title-icon="fa-plug"
-      endpoints="endpoints"
+      endpoints="endpoints" table-key="templates"
```

Comment:
Using the same `table-key` for this component and the `template-list` component will make them share their filter.

```suggestion
      endpoints="endpoints" table-key="endpoints"
```